src/main/java/com/actionlistapi/model/ActionRequest.java

```diff
@@ -0,0 +1,158 @@
+package com.actionlistapi.model;
+
+import java.util.ArrayList;
+
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+
+import com.actionlistapi.pojo.ActionRequested;
+import com.actionlistapi.pojo.ProcessInstanceStatus;
+import com.actionlistapi.services.ActionRequestedConverter;
+import com.actionlistapi.services.ProcessInstanceStatusConverter;
+import com.actionlistapi.services.RouteNodeConverter;
+
+@Entity
+public class ActionRequest {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY,generator="id_seq")
```

Comment:
Using sequential IDs is a potential security vulnerability: https://www.owasp.org/index.php/Top_10_2007-Insecure_Direct_Object_Reference While it's not likely to cause a problem in this case, since access to Action Requests should be locked down to just the user who received it, it would be better to use a UUID generator.